[TASK] Add togglable edit/preview button

robbieaverill opened the pull request:
Suggested modification: add a togglable edit/preview button (like GitHub) to minimize the space used on the page.
- Hide and show the preview/editor areas
- Hide the buttons while in preview mode
- Float the button to the right

robbieaverill committed:
[TASK] Add togglable edit/preview button

* Hide and show the preview/editor areas
* Hide the buttons while in preview mode

robbieaverill commented:
Thanks @priithansen - I haven't updated the readme, probably worth doing

priithansen commented:
Hei @robbieaverill thank You for the improvements!
Sad thing is these days I don't find much time to make websites with Silverstripe and thus am not using the modules acitvely. If You would like to take over the ownership of this little module I would have no problem with transferring it over to You.

robbieaverill commented:
Hi @priithansen 

No need to do that! It doesn't seem like an overly active module so there'd be no harm in leaving it as it is.

I'll push an update for the readme now for you though.

Cheers,
Robbie

priithansen commented:
Sounds good to me.

robbieaverill commented:
Hi again @priithansen 

If you want to transfer it to me I'm happy to take it over as I'll probably be using this for some projects coming up.

No problem though. I am also happy to maintain a fork.

Cheers,
Robbie

priithansen commented:
@robbieaverill I think it would be a good idea so if anybody else wants to use it they don't have to choose between unmaintained official repo and maintained forked one :)

I will try to do the transfer in project settings then.

priithansen commented:
No luck "robbieaverill/silverstripe-markdowntextareafield already exists"

robbieaverill commented:
Just deleted my fork, try again

On Friday, 19 February 2016, Priit notifications@github.com wrote:

> No luck "robbieaverill/silverstripe-markdowntextareafield already exists"
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/priithansen/silverstripe-markdowntextareafield/pull/4#issuecomment-186087108
> .

## 

_Robbie Averill_
LA PETITE MANOUCHE - NZ Gypsy Jazz Guitar Duo
Phone: +64 27 321 4758
contact@lapetitemanouche.com
www.lapetitemanouche.com

priithansen commented:
Request made

robbieaverill commented:
Cool. I will update the credits etc for you on Monday. Cheers! Robbie

On Friday, 19 February 2016, Priit notifications@github.com wrote:

> Request made
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/priithansen/silverstripe-markdowntextareafield/pull/4#issuecomment-186126866
> .

## 

_Robbie Averill_
LA PETITE MANOUCHE - NZ Gypsy Jazz Guitar Duo
Phone: +64 27 321 4758
contact@lapetitemanouche.com
www.lapetitemanouche.com

robbieaverill commented:
Hey @priithansen - I've added you to the readme etc for authorship - would you mind taking your Packagist repository down? I've added the original path as a `replace`ment in the `composer.json`, but with yours published still it will install an old version of the repo when requiring it with composer.

priithansen commented:
Hei deleted it. Didn't even remember I had an account there. Sorry about that.

robbieaverill commented:
No worries! Thanks

On Sunday, 21 February 2016, Priit notifications@github.com wrote:

> Hei deleted it. Didn't even remember I had an account there. Sorry about
> that.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/robbieaverill/silverstripe-markdowntextareafield/pull/4#issuecomment-186685143
> .

## 

_Robbie Averill_
LA PETITE MANOUCHE - NZ Gypsy Jazz Guitar Duo
Phone: +64 27 321 4758
contact@lapetitemanouche.com
www.lapetitemanouche.com
